analysis.py

```diff
@@ -0,0 +1,44 @@
+import json
+class Analysis:
+    def __init__(self, config):
+        self.log = self.loadJSON(config["logfile"])
```

Comment:
Reading the logfile causes a break if there is more than one simulation log in the file. This could be handled by reading only the last simulation log.